packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1726,13 +1762,16 @@ namespace Ifpack2 {
         cudaProfilerStart();
 #endif
 	const local_ordinal_type blocksize = internal_vector_values.extent(1);
-	const local_ordinal_type team_size = ExtractAndFactorizeTridiagsDefaultModeAndAlgo<typename execution_space::memory_space>::
+	const local_ordinal_type team_size = 
+	  ExtractAndFactorizeTridiagsDefaultModeAndAlgo<typename execution_space::memory_space>::
```

Comment:
It's a bit funny that this class takes a memory space, rather than an execution space.  Could it give different results for `CudaSpace` vs. `CudaUVMSpace`?  If not, then it should take an execution space (e.g., `Cuda`).

Reply:
I put template specialization for HostSpace, CudaSpace and CudaUVMSpace. I don't want to put specialization based on the execution space of OpenMP and Serial specialization.  One reason is that sometimes OpenMP or Serial space is not enabled and I need to put another macro to wrap the specialization. HostSpace is always defined. When Cuda is enabled, I just take care of CudaSpace and CudaUVMSpace only.